app/main/api/handlers/encryptWallet.ts

```diff
@@ -33,15 +34,17 @@ import { config } from "app/common/config"
  * @param params
  * @returns {Promise<void>}
  */
-export default async function encryptWallet(system: AppStateS & WalletStorageS, params: any):
-  Promise<JsonSerializable> {
+export default async function encryptWallet(
+  system: AppStateS & WalletStorageS & AppStorageS, params: any
+): Promise<JsonSerializable> {
 
   return parseParams(params)
     .then(inject(getUnconsolidatedData, system.appStateManager))
     .then(newWallet)
     .then(inject(newWalletStorage, system.storageFactory))
     .then(storeWallet)
     .then(inject(replaceWallet, system))
+    .then(inject(storePlainInfo, system))
```

Comment:
Again, I have doubts about not reverting actions when errors occur in this promise chain, but that's to be discussed in issue #358, so :+1: